src/encoded/types/__init__.py

```diff
@@ -96,6 +96,41 @@ class Enzyme(Item):
     embedded_list = Item.embedded_list + ['enzyme_source.title']
 
 
+@collection(
+    name='experiment-types',
+    unique_key='experiment_type:experiment_name',
+    properties={
```

Comment:
I think you might want to add a lookup_key as well for 'title' so that it can be used as the value in spreadsheets for submissions.  Otherwise submitters will need to use the experiment_name form or a uuid (or alias if you decide to add the mixin).